docs/getting-started/server/guide-vscode.md

```diff
@@ -0,0 +1,267 @@
+
+    ![Select container configuration](select-container-configuration.png)
+
+3.  Wait for the container to build and start. The first time you open the container, it will take a
+    few minutes to build.
+
+4.  Once the container has finished building for the first time, you will be greeted with the
+    `postCreateCommand` script, which will walk you through generating and applying any additional
+    secrets or certificates required for the server to run.
+
+5.  Follow the on-screen prompts to finish setting up the server. Then, proceed to
+    [Build and Run the Server](#build-and-run-the-server).
+
+## Build and Run the Server
+
+You are now ready to build and run your development server.
+
+1.  Open a new terminal window in the root of the repository.
+2.  Restore the nuget packages required for the Identity service:
+
+    ```bash
+    cd src/Identity
+    dotnet restore
+    ```
+
+3.  Start the Identity service:
+
+    ```bash
+    dotnet run
+    ```
+
+4.  Test that the Identity service is alive by navigating to
+    [http://localhost:33656/.well-known/openid-configuration](http://localhost:33656/.well-known/openid-configuration)
+5.  In another terminal window, restore the nuget packages required for the Api service:
+
+    ```bash
+    cd src/Api
+    dotnet restore
+    ```
+
+6.  Start the Api Service:
+
+    ```bash
+    dotnet run
+    ```
+
+7.  Test that the Api service is alive by navigating to
+    [http://localhost:4000/alive](http://localhost:4000/alive)
+8.  Connect a client to your local server by configuring the client’s Api and Identity endpoints.
+    Refer to
+    [https://bitwarden.com/help/article/change-client-environment/](https://bitwarden.com/help/article/change-client-environment/)
+    and the instructions for each client in the Contributing Documentation.
+
+:::info
+
+If you cannot connect to the Api or Identity projects, check the terminal output to confirm the
+ports they are running on.
+
+:::
+
+:::note
+
+We recommend continuing with the [Web Vault](../clients/web-vault/index.mdx) afterwards, since many
+administrative operations can only be performed in it.
+
+:::
+
+## Debugging
+
+You can use the debugger in Visual Studio Code to debug the server by following these steps:
+
+1. Click the "Debug and Run" button in the left sidebar of Visual Studio Code.
+
+2. Select the desired debug configuration from the dropdown menu.
+
+3. Click the "Start Debugging" button to start the debugger.
+
+## Managing and updating the Dev Containers
+
+After you’ve run the deployed the Dev Containers, you can use the
+[Docker Dashboard](https://docs.docker.com/desktop/dashboard/) or `docker` CLI to manage your
+containers. You should see your containers running under the `bitwarden_common` group.
+
+:::caution
+
+Changing `MSSQL_SA_PASSWORD` variable after first running the Dev Containers will require a
+re-creation of the storage volume.
+
+**Warning: this will delete your development database.**
```

Comment:
There's a `warn` decorator you can use here, but not sure if it works within a surrounding block.

Reply:
I updated the language and formatting for this process to hopefully make it more clear.